src/Language/PureScript/Constants/Prelude.hs

```diff
@@ -273,6 +273,9 @@ pattern DataSymbol = ModuleName "Data.Symbol"
 pattern IsSymbol :: Qualified (ProperName 'ClassName)
 pattern IsSymbol = Qualified (Just DataSymbol) (ProperName "IsSymbol")
 
+pattern IsNat :: Qualified (ProperName 'ClassName)
+pattern IsNat = Qualified (Just DataSymbol) (ProperName "IsNat")
```

Comment:
Should the module be `DataSymbol`? Or `DataNat`?

Reply:
Good catch! Thanks